core/src/main/java/software/amazon/awssdk/retry/conditions/MaxNumberOfRetriesCondition.java

```diff
@@ -13,9 +13,11 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.retry.v2;
+package software.amazon.awssdk.retry.conditions;
 
-import static software.amazon.awssdk.util.ValidationUtils.assertIsPositive;
+import static software.amazon.awssdk.utils.NumericUtils.assertIsPositive;
```

Comment:
Should this be in `Validate` with our other validation methods?

Reply:
Maybe? We had this NumericUtils class that already had a few validations for number types. Makes sense to me that NumericUtils should be used here but not a big deal to me either way.

Reply:
Validate already has some stuff for different types. We should decide on one place. Should we move string validation to StringUtils?